resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/config.scala

```diff
@@ -545,4 +545,36 @@ package object config extends Logging {
       resolvedURL
     }
   }
+
+  private[spark] val KUBERNETES_CLIENT_WATCH_RECONNECT_INTERVAL_SYSTEM_PROPERTY =
+    ConfigBuilder("spark.kubernetes.client.watch.reconnectInterval")
+      .doc("spark kubernetes client watch reconnectInterval," +
+        "which is relevant to KUBERNETES_WATCH_RECONNECT_INTERVAL_SYSTEM_PROPERTY" +
+        " in io.fabric8.kubernetes.client.Config.")
+      .stringConf
```

Comment:
Use `intConf` with default values everywhere.

Reply:
Actually `timeConf`might make some sense in some of these cases. I think these should have default values that are specified in the config entries and not in the main logic.

Reply:
Done